apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -47,55 +49,44 @@ defmodule OMG.Eth.RootChain do
     )
   end
 
-  def start_deposit_exit(deposit_positon, token, value, from, contract \\ nil, opts \\ []) do
-    defaults = @tx_defaults |> Keyword.put(:gas, 1_000_000)
-    opts = defaults |> Keyword.merge(opts)
-
-    contract = contract || from_hex(Application.fetch_env!(:omg_eth, :contract_addr))
-
-    Eth.contract_transact(
-      from,
-      contract,
-      "startDepositExit(uint256,address,uint256)",
-      [deposit_positon, token, value],
-      opts
-    )
-  end
+  def start_exit(outputId, txbytes, proof, from, contract \\ nil, opts \\ []) do
+    defaults =
+      @tx_defaults
+      |> Keyword.put(:gas, 1_000_000)
+      # standardExitBond = 31415926535 wei
+      |> Keyword.put(:value, 31_415_926_535)
 
-  def start_exit(utxo_position, txbytes, proof, sigs, from, contract \\ nil, opts \\ []) do
-    defaults = @tx_defaults |> Keyword.put(:gas, 1_000_000)
     opts = defaults |> Keyword.merge(opts)
 
     contract = contract || from_hex(Application.fetch_env!(:omg_eth, :contract_addr))
 
     Eth.contract_transact(
       from,
       contract,
-      "startExit(uint256,bytes,bytes,bytes)",
-      [utxo_position, txbytes, proof, sigs],
+      "startStandardExit(uint192,bytes,bytes)",
+      [outputId, txbytes, proof],
       opts
     )
   end
 
-  def deposit(value, from, contract \\ nil, opts \\ []) do
-    defaults = @tx_defaults |> Keyword.put(:gas, 80_000)
+  def deposit(tx, value, from, contract \\ nil, opts \\ []) do
```

Comment:
I might be wrong here, it looks like tx (= rlp-encoded tx_bytes) contains output which has `value` and `from` information. Right :question: 
Duplicating this params we risk they are unmatched and inconsistent data is passed to the contract.